### DISCUSSION TITLE
ecore-glsp workspace .ui-schemas directory creation?

### DISCUSSION THREAD

**Discussion by GaryKopp** [eclipse-emfcloud/emfcloud#92] [All other] [answered]
Is there a suggested method of creating/designing the JSON files in ecore-glsp's .ui-schemas directory for an arbitrary model? Actually, as I look at these files in the example it's not clear to me exactly what they're supposed to represent and in exactly what form. Are there some rules or guidelines for this?

**Comment by planger** [comment, marked as answer]
A UI schema defines the general layout of the form. See https://jsonforms.io/docs/uischema/uischema for more details.
